aces-nifi-processors/src/main/java/net/acesinc/nifi/processors/sockets/SocketIOProcessor.java

```diff
@@ -321,18 +333,13 @@ public void call(Object... objects) {
                 logger.debug("Socket " + uri.toString() + " disconnected...");
             }
             //Socket event for reconnection attempts.
-        }).on(Socket.EVENT_RECONNECT_ATTEMPT, new Emitter.Listener() {
```

Comment:
@chuckgregoryACES So, I take it from these changes that they likely removed some of these events out of the API? These are the main second-half to me of your core changes on this commit.